peer.go

```diff
@@ -738,6 +743,7 @@ func (ms *msgStream) Stop() {
 func (ms *msgStream) msgConsumer() {
 	defer ms.wg.Done()
 	defer peerLog.Tracef(ms.stopMsg)
```

Comment:
not a blocker, but this should prob be deferred after printing the start message